pkg/features/features.go

```diff
@@ -108,4 +108,9 @@ const (
 	// owner: @vanjiii
 	// alpha: v1.31.0
 	DenyInvalidExtensionResources featuregate.Feature = "DenyInvalidExtensionResources"
+
+	// CopyEtcdBackupsDuringMigration enables the copy of etcd backups between object stores belinging to different seds during control plane migration
+	// owner: @plkokanov
+	// alphsa: v1.36.0
+	CopyEtcdBackupsDuringMigration featuregate.Feature = "CopyEtcdBackupBuckets"
```

Comment:
Let's keep the actual feature gate names consistent with the constant names
```suggestion
	CopyEtcdBackupsDuringMigration featuregate.Feature = "CopyEtcdBackupsDuringMigration"
```